tests/test_api.py

```diff
@@ -30,139 +32,80 @@ def bliss_client(cwd, cfg):
 @pytest.fixture(scope="class")
 def cached_data_path_api(bliss_client):
     bliss_client.cached_data_path = bliss_client.cwd + "/data/cached_dataset"
-    bliss_client.generate(n_batches=3, batch_size=8, max_images_per_file=16)
+    bliss_client.generate(n_batches=2, batch_size=5, max_images_per_file=5)
     return bliss_client.cached_data_path
 
 
-def download_pretrained_weights(bliss_client, cfg, filename):
-    # Only test downloading pretrained weights (from Git LFS) if run via GitHub Actions
-    if os.environ.get("GITHUB_TOKEN") is None:
-        # Run locally, so use pretrained weights from local BLISS_HOME
-        # Copy pretrained weights to {cwd}/data/pretrained_models
-        local_pretrained_weights_path = Path(cfg.paths.data) / "pretrained_models/sdss.pt"
-        test_pretrained_weights_path = Path(bliss_client.cwd) / f"data/pretrained_models/{filename}"
-        test_pretrained_weights_path.parent.mkdir(parents=True, exist_ok=True)
-        shutil.copy(str(local_pretrained_weights_path), str(test_pretrained_weights_path))
-    else:
-        bliss_client.load_pretrained_weights_for_survey(
-            survey="sdss",
-            filename=filename,
-            request_headers={"Authorization": f"Bearer {os.environ['GITHUB_TOKEN']}"},
-        )
-    not_found_err_msg = (
-        "pretrained weights "
-        + f"{bliss_client.cwd}/data/pretrained_models/{filename} "
-        + "not found"
-    )
-    assert Path(
-        bliss_client.cwd + f"/data/pretrained_models/{filename}"
-    ).exists(), not_found_err_msg
-
-
 @pytest.fixture(scope="class")
-def pretrained_weights_filename(bliss_client, cfg):
-    filename = "sdss_pretrained.pt"
-    download_pretrained_weights(bliss_client, cfg, filename)
-    return filename
-
-
-@pytest.fixture(scope="class")
-def weight_save_path(bliss_client, pretrained_weights_filename):
+def weight_save_path(bliss_client, cfg):
+    """Train model for 1 epoch and return path to saved model."""
     weight_save_path = "tutorial_encoder/0_fixture.pt"
     bliss_client.train_on_cached_data(
         weight_save_path=weight_save_path,
         train_n_batches=1,
-        batch_size=8,
+        batch_size=5,
```

Comment:
8 here?

Reply:
Keeping this at 5 as per the other comment.